### PR TITLE
Print tox minversion+requires with --print-deps-to

### PR DESCRIPTION
**Pull request by hroncok**
Fixes https://github.com/fedora-python/tox-current-env/issues/39

### CODE REVIEW

File: src/tox_current_env/hooks.py [hunk 1]
```diff
@@ -197,6 +197,15 @@ def tox_testenv_install_deps(venv, action):
         return True
 
 
+def tox_dependencies(config):
+    """Get dependencies of tox itself, 'minversion' and 'requires' config options"""
+    # config does not have this attribute until tox 3.22:
+    deps = getattr(config, "requires", [])
+    if config.minversion is not None:
+        deps.insert(0, f"tox >= {config.minversion}")
+    return deps
```
Comment (encukou) [inline src/tox_current_env/hooks.py:206]: Would this be clearer as a generator?
```suggestion
    if config.minversion is not None:
        yield f"tox >= {config.minversion}"
    # config does not have the "requires" attribute until tox 3.22:
    yield from getattr(config, "requires", [])
```

File: README.rst [hunk 2]
```diff
@@ -207,6 +207,44 @@ use the ``TOX_TESTENV_PASSENV`` environment variable.
...
+The tested projects can specify the
+`minimal tox version <https://tox.readthedocs.io/en/latest/config.html#conf-minversion>`_
+and/or
+`additional requires <https://tox.readthedocs.io/en/latest/config.html#conf-requires>`_
+needed in the environment where ``tox`` is installed.
+Normally, ``tox`` uses *provisioning* when such requirements are not met.
+It creates a virtual environment,
+installs (a newer version of) ``tox`` and the missing packages
+into that environment and proxies all ``tox`` invocations trough that.
+Unfortunately , this is undesired for ``tox-current-env``.
```
Comment (encukou) [inline README.rst:222]:
```suggestion
Unfortunately, this is undesired for ``tox-current-env``.
```

**Review by encukou — COMMENTED** [review]
This looks good! Fix a typo and it's ready to ship :)
I also have a minor style suggestion:

File: src/tox_current_env/hooks.py (hunk 1 above)
Comment (hroncok) [reply src/tox_current_env/hooks.py:206]: Awesome, yes.

**Pull request merged**